BuildResidentialHPXML/measure.rb

```diff
@@ -2635,6 +2637,297 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
+    arg.setUnits('#')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeIntegerArgument('electric_panel_breaker_spaces_rated_total', false)
+    arg.setDisplayName('Electric Panel: Breaker Spaces Rated Total')
+    arg.setDescription('The rated total number of breaker spaces on the electric panel.')
+    arg.setUnits('#')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('electric_panel_load_heating_system_power_rating', false)
+    arg.setDisplayName('Electric Panel: Heating System Power Rating')
+    arg.setDescription("Specifies the panel load heating system power. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#service-feeders'>Panel Loads</a>) is used.")
+    arg.setUnits('W')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('electric_panel_load_heating_system_new_load', false)
+    arg.setDisplayName('Electric Panel: Heating System New Load')
+    arg.setDescription("Whether the heating system is a new panel load addition to an existing service panel. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#service-feeders'>Service Feeders</a>) is used.")
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('electric_panel_load_cooling_system_power_rating', false)
+    arg.setDisplayName('Electric Panel: Cooling System Power Rating')
+    arg.setDescription("Specifies the panel load cooling system power rating. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#service-feeders'>Service Feeders</a>) is used.")
+    arg.setUnits('W')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('electric_panel_load_cooling_system_new_load', false)
+    arg.setDisplayName('Electric Panel: Cooling System New Load')
+    arg.setDescription("Whether the cooling system is a new panel load addition to an existing service panel. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#service-feeders'>Service Feeders</a>) is used.")
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('electric_panel_load_heat_pump_heating_power_rating', false)
+    arg.setDisplayName('Electric Panel: Heat Pump Heating Power Rating')
+    arg.setDescription("Specifies the panel load heating power rating. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#service-feeders'>Service Feeders</a>) is used.")
+    arg.setUnits('W')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('electric_panel_load_heat_pump_cooling_power_rating', false)
+    arg.setDisplayName('Electric Panel: Heat Pump Cooling Power Rating')
+    arg.setDescription("Specifies the panel load heat pump cooling power. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#service-feeders'>Service Feeders</a>) is used.")
+    arg.setUnits('W')
+    args << arg
```

Comment:
Why are there separate heating vs cooling power ratings for a HP? Seems like it should only be one input.

Reply:
I don't really follow. If you are looking at some heat pump product in the market, won't you use a single heat pump power rating in the NEC calculation?

Reply:
This explanation/assumption makes sense to me. I'll condense the separate arguments into one.

Reply:
https://github.com/NREL/OpenStudio-HPXML/pull/1946/commits/d2f0cc668be96a5386132f817c31e9a5c69b69fb